pwa/app/components/tba/rpDot.tsx

```diff
@@ -0,0 +1,65 @@
+import { Match } from '~/api/v3';
+import {
+  Tooltip,
+  TooltipContent,
+  TooltipProvider,
+  TooltipTrigger,
+} from '~/components/ui/tooltip';
+import {
+  RANKING_POINT_LABELS,
+  getBonusRankingPoints,
+} from '~/lib/rankingPoints';
+import { cn } from '~/lib/utils';
+
+function RpDot({
+  rpIndex,
+  tooltipText,
+}: {
+  rpIndex: number;
+  tooltipText: string;
+}) {
+  return (
+    <TooltipProvider delayDuration={100}>
+      <Tooltip>
+        <TooltipTrigger asChild>
+          <svg
+            className={cn('h-[4px] absolute top-[2px] left-[3px] w-[4px]', {
+              'ml-0': rpIndex === 0,
+              'ml-[6px]': rpIndex === 1,
```

Comment:
Does this scale to 3 RP dots in 2025?

Reply:
Yes, just not implemented yet, will do when we get sample data.